flink-table/flink-table-common/src/main/java/org/apache/flink/table/descriptors/Schema.java

```diff
@@ -183,11 +250,26 @@ public Schema proctime() {
 		return this;
 	}
 
+	/**
+	 * Specifies the computed column as a processing-time attribute.
+	 *
+	 * <p>E.g. proctime("pt")
+	 */
+	public Schema proctime(String proctimeAttribute) {
+		addField(proctimeAttribute, "TIMESTAMP(3)", "PROCTIME()");
+		tableSchema.get(proctimeAttribute).put(SCHEMA_PROCTIME, "true");
```

Comment:
Is there any problems if we don't add this property?

Reply:
The old `field(...).proctime()` api use this to identify a proctime field. We add this for back-compatibility. And we also use this property to deserialize/serialize TableSchema in DescriptorProperties.